services/src/main/java/org/keycloak/services/resources/LoginActionsService.java

```diff
@@ -868,13 +868,13 @@ public Response processConsent(final MultivaluedMap<String, String> formData) {
 
         for (String clientScopeId : authSession.getClientScopes()) {
             ClientScopeModel clientScope = KeycloakModelUtils.findClientScopeById(realm, client, clientScopeId);
-            if (clientScope != null && clientScope.isDisplayOnConsentScreen()) {
```

Comment:
I changed this line because client scopes which are not displayed on consent screen should not be logged as warning.